### ISSUE TITLE
codeformat.py doesn't fix spaces in function calls/declarations

### ISSUE THREAD

**Issue by stinos** [micropython/micropython#5774]
Code like
```
int gettimeofday( struct timeval *tp, struct timezone *tz );
ZeroMemory(&ft, sizeof(ft)     );
```
remains unchanged (Windows, Uncrustify-0.70.1_f)

**Comment by dpgeorge** [comment]
See #5777 for a fix.

**Comment by dpgeorge** [comment]
Fixed by feb25775851ba0c04b8d1013716f442258879d9c
